tests/build_tests/super_errors/expected/jsx_custom_component_children.res.expected

```diff
@@ -9,6 +9,4 @@
 
   This has type: [1;31mfloat[0m
   But children passed to this component must be of type:
-    [1;33mReact.element[0m [2m(defined as[0m [1;33mJsx.element[0m[2m)[0m
-  
-  In JSX, all content must be JSX elements. You can convert float to a JSX element with [1;33mReact.float[0m.
```

Comment:
We are losing the improved error messages here, so it seems that some adjustment to the error handling logic is missing. /cc @zth